scripts/veloxbench/veloxbench/implemented_benchmarks.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import conbench.runner
+
+from .cpp_micro_benchmarks import LocalCppMicroBenchmarks
+from ._benchmark import Benchmark
+
+
+@conbench.runner.register_benchmark
+class RecordCppMicroBenchmarks(LocalCppMicroBenchmarks, Benchmark):
```

Comment:
who ends up using this class?

Reply:
In order to be able to use the CLI to do `conbench some-benchmark ...`, you need to define a class that inherits from `conbench.runner.Benchmark` and decorate it with `@conbench.runner.register_benchmark`. That's all this file is doing.

With some [work](https://github.com/conbench/conbench/tree/main/benchadapt) going on right now in the Conbench repo, in the future it should be a lot easier and less confusing to run Conbench!